soc/xtensa/intel_adsp/common/trace_out.c

```diff
@@ -40,9 +43,16 @@ int arch_printk_char_out(int c)
 	return 0;
 }
 
-void soc_trace_init(void)
+int soc_trace_init(void)
```

Comment:
Changing the return type seems not relevant in this patch - maybe do this only in the next one?